project/app/api/models.py

```diff
@@ -148,3 +146,49 @@ def check_valid_subid(cls, value):
     def check_sha_len(cls, value):
         assert len(value) == 128
         return value
+
+
+class LineGraphRequest(BaseModel):
+    """Model that handles LineGraph Request to API
+    """
```

Comment:
Can you move the closing quotes on each of these classes up a line, so they're all 3 single-line docstrings?